versioned_docs/version-1/05-gitlab/01-getting-started.md

```diff
@@ -0,0 +1,53 @@
+# Getting started
+
+This guide will demonstrate how to setup build and test automation for your Unity project hosted on gitlab using gitlab-ci.
```

Comment:
Actually losing all version information from the docs is not desirable.

Could you make sure git recognises that it's the same file as before, by making changes in one commit and doing the rename in another?

Reply:
oh, hmm, not quite sure how to achieve this.  I used `yarn run docusaurus docs:version 1` command and replaced the copy of generated docs with the content of v1.

Reply:
From what I see, history looks fine. What you are looking at is an actual copy of the gitlab docs. V1 was only for github.

```diff
diff --git a/versioned_docs/version-1/03-docker/versions.md b/versioned_docs/version-1/03-docker/versions.md
new file mode 100644
index 0000000..6af101f
--- /dev/null
+++ b/versioned_docs/version-1/03-docker/versions.md
@@ -0,0 +1 @@
+[//]: # ( See src/pages/docs/docker/versions.tsx )
diff --git a/docs/04-github/v1/01-getting-started.md b/versioned_docs/version-1/04-github/01-getting-started.md
similarity index 100%
rename from docs/04-github/v1/01-getting-started.md
rename to versioned_docs/version-1/04-github/01-getting-started.md
diff --git a/docs/04-github/v1/02-activation.md b/versioned_docs/version-1/04-github/02-activation.md
similarity index 100%
rename from docs/04-github/v1/02-activation.md
rename to versioned_docs/version-1/04-github/02-activation.md
diff --git a/docs/04-github/v1/03-test-runner.md b/versioned_docs/version-1/04-github/03-test-runner.md
similarity index 100%
rename from docs/04-github/v1/03-test-runner.md
rename to versioned_docs/version-1/04-github/03-test-runner.md
diff --git a/docs/04-github/v1/04-builder.md b/versioned_docs/version-1/04-github/04-builder.md
similarity index 100%
rename from docs/04-github/v1/04-builder.md
rename to versioned_docs/version-1/04-github/04-builder.md
diff --git a/docs/04-github/v1/05-returning-a-license.md b/versioned_docs/version-1/04-github/05-returning-a-license.md
similarity index 100%
rename from docs/04-github/v1/05-returning-a-license.md
rename to versioned_docs/version-1/04-github/05-returning-a-license.md
diff --git a/docs/04-github/v1/06-deployment/android.md b/versioned_docs/version-1/04-github/06-deployment/android.md
similarity index 100%
rename from docs/04-github/v1/06-deployment/android.md
rename to versioned_docs/version-1/04-github/06-deployment/android.md
diff --git a/docs/04-github/v1/06-deployment/ios.md b/versioned_docs/version-1/04-github/06-deployment/ios.md
similarity index 100%
rename from docs/04-github/v1/06-deployment/ios.md
rename to versioned_docs/version-1/04-github/06-deployment/ios.md
diff --git a/versioned_docs/version-1/05-gitlab/01-getting-started.md b/versioned_docs/version-1/05-gitlab/01-getting-started.md
new file mode 100644
index 0000000..8db0260
--- /dev/null
+++ b/versioned_docs/version-1/05-gitlab/01-getting-started.md
```

The commit already does a _rename_. Something we could do is actually to remove `gitlab` and `intro` pages from v1 docs, but then I think it would become even more misleading.

Reply:
git usually recognises move operations. But if you have both files at the same time it will not recognise it as a move, even if you delete the file after that (depending on local git configuration).

Usually this only ever happens with 1~20 files. What I usually end up doing in a situation like yours is to generate the structure to see what the filenames should be and then on a file by file basis
- Copy the new filename. 
- Delete that new file. 
- Rename the original file to the copied file name

Then commit, and then make changes. Git recognises it's the same file until you make changes that affect more than a percentage of the file.

Reply:
Ah ok, so it's just because the GitLab files for v1 never existed before. Check.

Reply:
Considering this as resolved 👍